### ISSUE TITLE
Daily Digest

### ISSUE THREAD

**Issue by jacquescrocker** [OpenHunting/openhunt#16]
allow people to sign up to receive the top products of the day via email

**Comment by nblackburn** [comment]
+1

**Comment by nealrs** [comment]
@jacquescrocker i'd be happy to lend a hand with this - templates / ideas / etc. I run the [Devpost](http://devpost.com) weekly newsletter + [Git@Me](http://gitat.me) (my own about open source projects)

**Comment by jacquescrocker** [comment]
@nealrs thanks!

**Comment by jacquescrocker** [comment]
via hn https://news.ycombinator.com/item?id=10762846

"Don't do daily... or atleast let people sign up for daily/weekly/monthly. The PH daily emails were too much. I already get enough crap I don't read, it just makes me resent the site if I feel they are spamming me too much."

**Comment by jacquescrocker** [comment]
email subscription (for daily) is now deployed

we'll do the daily one first, then add an option for weekly later

**Comment by mhurwi** [comment]
I agree. Weekly should be easy after daily is setup, it's just about gathering the appropriate buckets for the week. 

The daily digest preparation is done, will be moving on to the 'send' part today.
